pyi.py

```diff
@@ -693,9 +693,12 @@ def _analyse_union(members: Sequence[ast.expr]) -> UnionAnalysis:
         "sys.version",
         "sys.version_info",
         "sys.winver",
+        "_typeshed.sentinel",
     }
 )
 
+_ALLOWED_SIMPLE_ATTRIBUTES_IN_DEFAULTS = frozenset({"sentinel"})
```

Comment:
I don't _love_ this, since `sentinel` is a pretty common variable name, and it could be imported from other places. But I also understand that forcing people to use `_typeshed.sentinel` everywhere would be annoyingly verbose, so I'm fine with it for now.

It's on my to-do list to teach flake8-pyi about how imports work, so that we'd be able to figure out whether `sentinel` had actually been imported from `_typeshed` or not. But it'll involve a fair bit of refactoring to do that...

Reply:
Indeed, proper resolving would be great as it would also allow constructs like `from _typeshed import sentinel as _sentinel`.